source/server/connection_handler_impl.cc

```diff
@@ -13,7 +14,9 @@ namespace Envoy {
 namespace Server {
 
 ConnectionHandlerImpl::ConnectionHandlerImpl(spdlog::logger& logger, Api::ApiPtr&& api)
-    : logger_(logger), api_(std::move(api)), dispatcher_(api_->allocateDispatcher()) {}
+    : logger_(logger), api_(std::move(api)), dispatcher_(api_->allocateDispatcher()) {
+  UNREFERENCED_PARAMETER(logger_);
```

Comment:
Why is this needed? To handle the GLOG_ON case, which case I assume that logger_ is an unused member, rather than unused parameter.